cognite/neat/rules/models/_rules/dms_schema.py

```diff
@@ -144,3 +165,176 @@ def validate(self) -> list[DMSSchemaError]:
                     errors.add(DuplicatedViewInDataModelError(referred_by=model.as_id(), view=view_id))
 
         return list(errors)
+
+
+@dataclass
+class PipelineSchema(DMSSchema):
+    transformations: TransformationWriteList = field(default_factory=lambda: TransformationWriteList([]))
+    databases: DatabaseWriteList = field(default_factory=lambda: DatabaseWriteList([]))
+    raw_tables: RawTableWriteList = field(default_factory=lambda: RawTableWriteList([]))
+
+    _SQL_TYPE_BY_PROPERTY_TYPE: ClassVar[dict[type[dm.PropertyType], str]] = {
```

Comment:
This should be moved under `cognite.neat.rules.models._rules._types._value` as part of `_DATA_TYPES`, eg:

{"name": "boolean", "python": bool, "GraphQL": "Boolean", "dms": Boolean, "raw":dm.Boolean},


and then similar to how we made `XSD_VALUE_TYPE_MAPPINGS` and `DMS_VALUE_TYPE_MAPPINGS`, `SQL_VALUE_TYPE_MAPPINGS` should be created.

In this way we keep thing contained at right place.